src/validation/error.messages.ts

```diff
@@ -176,6 +178,7 @@ export enum ErrorMessages {
   START_DATE_BEFORE_FILING_DATE = "Start date must be before or on the date of the update statement",
   CEASED_DATE_BEFORE_FILING_DATE = "Ceased date must be before or on the date of the update statement",
   RESIGNED_ON_BEFORE_FILING_DATE = "Resigned on date must be before or on the date of the update statement",
+  DATE_AFTER_MADE_UP_TO_DATE = "The date you provide must be on or before %s",
```

Comment:
JIRA ticket needs updating to match this implementation - currently a bit unclear:

_Error message: The date you provide must be on or before ‘the entity's update statement date’ 
The entity's update statement date’ shall we replace this with the numeric date._